src/test/java/greencity/service/impl/PlaceServiceImplTest.java

```diff
@@ -73,54 +156,22 @@
 
     @Test
     public void save() {
```

Comment:
the naming of the test methods are incorrect.
not just "save" but more specific